package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "sort-keys",
-	"version": "4.0.0",
+	"version": "5.0.0",
```

Comment:
Don't bump version in a PR. It's up to the maintainers to choose what version and when to update it.